common/copr_common/request.py

```diff
@@ -65,6 +65,17 @@ def _send_request(self, url, method, data=None, **kwargs):
 
         headers = kwargs.get("headers", None) or {}
         headers["User-Agent"] = self.user_agent
+
+        if files:
+            # For each file object in dictionary we reset the pointer to start
+            # Check that we are actually dealing with file objects before trying anything.
+            for file in files.values():
+                if hasattr(file, 'seek') and hasattr(file, 'tell'):
+                    try:
+                        file.seek(0)
```

Comment:
Could we just seek_set 0 in `except RequestRetryError as ex` below, inside `_send_request_repeatedly`?

We could avoid those hasattr() checks likely, and simply seek -> if it tracebacks, it's OK -- we have to seek, otherwise we are risking just too much.